### PR TITLE
Fix time stamp output from time_vector.py so it is Ecl compatible

### PR DESCRIPTION
**Pull request by vkip**
Minor update so time stamp adheres to HH:MM:SS[.SSSS]  (+ some formatting improvements)

### CODE REVIEW

**Comment by bska** [comment]
jenkins build this please

File: python/opm/tools/time_vector.py [hunk 1]
```diff
@@ -9,65 +9,71 @@
...
+    "JUN": 6,
+    "JLY": 7,
+    "JUL": 7,
+    "AUG": 8,
+    "SEP": 9,
+    "OCT": 10,
+    "OKT": 10,
+    "NOV": 11,
+    "DEC": 12,
+    "DES": 12,
```
Comment (blattms) [inline python/opm/tools/time_vector.py:28]: I know, you fixed only indentation here and this was like this in the original version. But while you are at it: Shouldn't this read "DEC"?

**Review by blattms — APPROVED** [review]
Looks good to mw. I will merge this now.

Please take a look at my comment below, whether DES should be DEC. maybe we should fix this in another PR:

File: python/opm/tools/time_vector.py (hunk 1 above)
Comment (bska) [reply python/opm/tools/time_vector.py:28]: > shouldn't this read "DEC"?

You'll notice that there is a `DEC` line just above the `DES` line.  For what it's worth, ECLIPSE ([and Flow](https://github.com/OPM/opm-common/blob/120ef995109afec5757352fc0e30c2c36ac6f172/src/opm/common/utility/TimeService.cpp#L33)) allows a Scandinavian spelling of certain month name abbreviations.  This alternate spelling applies to [MAY](https://github.com/OPM/opm-common/blob/120ef995109afec5757352fc0e30c2c36ac6f172/src/opm/common/utility/TimeService.cpp#L38-L39), [OCT](https://github.com/OPM/opm-common/blob/120ef995109afec5757352fc0e30c2c36ac6f172/src/opm/common/utility/TimeService.cpp#L45-L46), and [DEC](https://github.com/OPM/opm-common/blob/120ef995109afec5757352fc0e30c2c36ac6f172/src/opm/common/utility/TimeService.cpp#L48-L49).

We use the English spelling when translating from month indices to month names, however.

**Pull request merged**